lang/fpc/Portfile

```diff
@@ -265,6 +271,13 @@ if {${subport} eq "${name}"} {
                         PREFIX=${destroot}${fpcbasepath}
     build.args          OPT="-ap -v0 ${linkerOptions}" CPU_TARGET=${cpuTarget} UTILS=1
 
+    # codesign for older systems does not have a --remove-signature option
+    # work around: replace codesign by dummy.
+    # os.major = 15 is macOS = 10.11 (El Capitan)
+    if {${os.major} <= 15} {
+        build.args-append   CODESIGN=/usr/bin/true
```

Comment:
I have thrown it away on 10.6, as you have seen, but perhaps this is just as fine.